cylc/flow/scripts/lint.py

```diff
@@ -469,6 +467,10 @@ def main(parser: COP, options: 'Values', *targets) -> None:
     else:
         targets = (str(Path.cwd()),)
 
+    # make sure the targets are all src/run directories
+    for target in targets:
+        check_flow_file(target)
```

Comment:
Without this the linter will attempt to recursively glob for Cylc files even in directories which are not workflow directories.

Discovered when I typed `cylc lint` in my home dir.